cli/create_test.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	vc "github.com/kata-containers/runtime/virtcontainers"
 	"github.com/kata-containers/runtime/virtcontainers/pkg/oci"
-	"github.com/kata-containers/runtime/virtcontainers/pkg/vcMock"
+	"github.com/kata-containers/runtime/virtcontainers/pkg/vcmock"
```

Comment:
Why did `golint` recommended this ? I thought Golang was all about camel case ?

Reply:
Here's the message:

```
virtcontainers/pkg/vcMock/container.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/mock.go:25:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/mock_test.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/pod.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/types.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/utils.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
```

Reply:
Okay, fair enough